### DISCUSSION TITLE
webdav插件安装在路由

### DISCUSSION THREAD

**Discussion by Evan-tc** [messense/aliyundrive-webdav#338] [Q&A] [answered]
### 问题描述

远程连接webdav在线看电视或者下载是不是会先缓冲（缓存）路由内存里？

### 重现步骤

远程连接webdav在线看电视或者下载是不是会先缓冲（缓存）路由闪存里？

### 版本

v1

### 运行平台

小米路由r4a千兆

### 日志

小米路由闪存只有16m，这是不是意味着没法在线用，因为我在线看视频无法加载出来，麻烦了。

**Comment by messense** [comment, marked as answer]
当然需要，可以调整 `--read-buffer-size`，默认是 10MB
